float64 back_l
float64 back_r

float64 left_b
float64 left_m
float64 left_f

float64 front_l
float64 front_r

float64 right_f
float64 right_m
float64 right_b